src/flows/parser_item_html_text_streaming_flow.py

```diff
@@ -133,26 +197,82 @@ def load(key: str):
     logger.info(f"deleting file: {str(key)}")
-    s3 = boto3.resource('s3')
+    s3 = boto3.resource("s3")
     s3.Object(bucket, key).delete()
 
 
-with Flow(FLOW_NAME, executor=LocalDaskExecutor(), schedule=get_interval_schedule(minutes=1440)) as flow:
+@task()
+def ordered_dataset_insert_transform() -> None:
+    """This task will incrementally insert new records into the article_content_ordered_live table.
+    This table was created manually from existing raw data as a starting point.
+    Query is built so that this task is itempotent.
+    """
+    logger = prefect.context.get("logger")
+    logger.info("Adding new records to article_content_ordered_live dataset...")
+    with sf.connect(**SNOWFLAKE_DEFAULT_DICT) as conn:
+        result = conn.execute_string(INSERT_ORDERED_SQL)
+        logger.info(f"Inserted {result[1].rowcount} rows...")
+
+
+@task()
+def weekly_reordering_transform() -> None:
+    """This task will perform a reordering of the article_content_ordered_live table weekly.
+    This involves a new table being swaped with the old.
+    Grants are cleaned up.
+    Logic makes it that this will only run on Saturday if the last_ordered date is older than a week.
+    A single run of this task updates the last _ordered date and prevents multiple runs.
+    """
+    logger = prefect.context.get("logger")
+    logger.info("Checking to see if reordering is needed...")
+    with sf.connect(**SNOWFLAKE_DEFAULT_DICT) as conn:
+        result = (
+            conn.cursor()
+            .execute(
+                f"""select datediff('hours', max(last_ordered), current_timestamp()) > 168 as ready_for_reordering
+   from {DATABASE_NAME}.{SCHEMA_NAME}.article_content_ordered_live;"""
+            )
+            .fetchone()
+        )
+        ready_for_reordering = result[0]
+    current_dt = pendulum.now("UTC")
+    # this should be Saturday
+    if current_dt.day_of_week == 6 and ready_for_reordering:
+        logger.info("Executing reordering of article_content_ordered_live dataset...")
+        with sf.connect(**SNOWFLAKE_DEFAULT_DICT) as conn:
+            conn.execute_string(WEEKLY_DEDUP_SQL)
+    else:
+        logger.info("Reordering not needed at this time...")
+
+
+with Flow(
+    FLOW_NAME,
+    executor=LocalDaskExecutor(),
+    schedule=get_interval_schedule(minutes=1440),
+) as flow:
     source_keys_results = get_source_keys()
     extract_results = extract.map(source_keys_results)
     transform_results = transform.map(extract_results)
     stage_results = stage(transform_results)
     load_results = load.map(stage_results)
-    cleanup.map(flatten([source_keys_results, stage_results])).set_upstream(load_results)
+    cleanup_task = cleanup.map(
+        flatten([source_keys_results, stage_results])
+    ).set_upstream(load_results)
+    ordered_dataset_insert = ordered_dataset_insert_transform().set_upstream(
+        cleanup_task
+    )
+    weekly_reordering_transform().set_upstream(ordered_dataset_insert)
```

Comment:
Why do you run the weekly order transform on a daily schedule?

Reply:
Actually that schedule is for the flow.  It looks like as some point someone disabled the daily scheduled in the Cloud UI and added an hourly schedule.  It seems like we want to keep that hourly schedule?  Should change the flow to hourly at 15 past the hour?